### PR TITLE
{tools}[FCC/21.05] Perl v5.32.0, DB v18.1.40, expat v2.2.9, ...

### PR DESCRIPTION
**Pull request by migueldiascosta**
(created using `eb --new-pr`)

requires ~#12863~

### CODE REVIEW

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**FAILED**
Build succeeded for 0 out of 8 (8 easyconfigs in total)
d28-3009c - Linux RHEL 8.3, AArch64, UNKNOWN (a64fx), Python 3.6.8
See https://gist.github.com/097111427a79cd62f826c90def010189 for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in total)
e31-1001c - Linux RHEL 8.3, AArch64, UNKNOWN (a64fx), Python 3.6.8
See https://gist.github.com/b8481972107b8e676bd9a49b2c828af0 for a full test report.

File: easybuild/easyconfigs/d/DB/DB-18.1.40-FCC-21.05.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+description = """Berkeley DB enables the development of custom data management
+ solutions, without the overhead traditionally associated with such custom
+ projects."""
+
+toolchain = {'name': 'FCC', 'version': '21.05'}
+
+# use Homebrew source mirror to allow auto-downloading source tarball
+# (Oracle website - https://www.oracle.com/technetwork/products/berkeleydb/downloads/ - requires reigstration)
+source_urls = ['https://bintray.com/homebrew/mirror/download_file?file_path=']
```
Comment (boegel) [inline easybuild/easyconfigs/d/DB/DB-18.1.40-FCC-21.05.eb:15]: This source URL needs to be fixed, see #12974

**Review by boegel — CHANGES_REQUESTED** [review]
OpenSSL should also be listed as an OS dep in the `DB` easyconfig

File: easybuild/easyconfigs/d/DB/DB-18.1.40-FCC-21.05.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/d/DB/DB-18.1.40-FCC-21.05.eb:15]: fixed in https://github.com/migueldiascosta/easybuild-easyconfigs/pull/34

**Comment by migueldiascosta** [comment]
replaced with https://github.com/easybuilders/easybuild-easyconfigs/pull/12997

**Pull request closed without merging**